dashboard/src2/pages/LoginSignup.vue

```diff
@@ -165,9 +176,21 @@ export default {
 			email: null,
 			password: null,
 			signupEmailSent: false,
-			resetPasswordEmailSent: false
+			resetPasswordEmailSent: false,
+			oauthEmailDomain: null
 		};
 	},
+	watch: {
+		email: {
+			handler: debounce(function (value) {
+				let domain = value.split('@').pop();
+				this.oauthEmailDomain =
```

Comment:
oauthEmailDomain can be a computed value

Reply:
@netchampfaris Didn't understand this. I'm trying to set the value of oauthEmailDomain and computed are readonly? 🤔

Reply:
The watcher is extracting the email domain from the user inputted `email` value right? This is the perfect use-case for computed value.